### ISSUE TITLE
Automatic reporting agent

### ISSUE THREAD

**Issue by Shikugawa** [SkyAPM/cpp2sky#17]
Seems that there is automatic reporting agent on skywalking-python. We'd like to introduce this functionality in the future. This project is currently `~0.1.0` focusing on integrating Envoy, but in the future.

https://github.com/apache/skywalking-python/blob/606a005fa046583d13922ec27f9b2b0fa58fe860/skywalking/trace/context.py#L166

**Comment by wu-sheng** [comment]
Yes, automatic reporting exists for every agent implementation, if this runs alone, no inside things like envoy.

**Comment by Shikugawa** [comment]
Sure. It will be treat as enhancement but no feature for `0.1.0`. Thanks.
